tests/dbz-twin/rac/kafka-consumer.py

```
@@ -139,18 +144,19 @@ def main():
                 sys.exit(1)
 
     # Wait for topics to appear, then subscribe
-    print(f"Waiting for topics: {LM_TOPIC}, {OLR_TOPIC}...", flush=True)
+    all_topics = [LM_TOPIC, OLR_TOPIC, OLR_LOB_TOPIC]
+    print(f"Waiting for topics: {', '.join(all_topics)}...", flush=True)
     for attempt in range(60):
```

Ruff (0.15.9)

[warning] Loop control variable `attempt` not used within loop body

(B007)